tests/test_planet_auth/unit/auth/auth_clients/oidc/test_oidc_request_authenticator.py

```diff
@@ -73,6 +73,18 @@ def setUp(self):
         self.tmp_dir = tempfile.TemporaryDirectory()
         self.tmp_dir_path = pathlib.Path(self.tmp_dir.name)
 
+    @staticmethod
+    def _filter_spdata(data: dict) -> dict:
+        # Storage providers are known to add "__" keys for their own use.
+        # These should not be considered part of the general data.
+        filtered_data = {
+            k: v
+            for k, v in data.items()
+            # if k not in [_SOPSAwareFilesystemObjectStorageProvider._STORAGE_TYPE_KEY]
```

Comment:
More commented out code without context. If this is a future nice-to-have as with the return True in `_do_sops` above, a comment as to what needs to change to reinstate this would be good; otherwise remove.